docs/source/run/parameters.rst

```diff
@@ -901,9 +901,36 @@ Parameters starting with ``lasers.`` apply to all laser pulses, parameters start
           Distance at which the laser pulse is focused (in the z direction, counted from laser initial position).
 
       * ``<laser name>.propagation_angle_yz`` (`float`) optional (default `0`)
-          Propagation angle of the pulse in the yz plane (0 is along the z axis)
+          Propagation angle of the pulse in the yz plane (0 is along z axis)
 
-      Option: ``from_file`` the laser is loaded from an openPMD file.
+      * ``<laser name>.STC_theta_xy`` (`float`) optional (default `pi/2`)
+          Direction of the linear spatial and angular chirp on xoy plane (0 is along x axis).
+
+      * ``<laser name>.beta`` (`float`) optional (default `0.`)
+          Angular dispersion (or angular chirp) at focus defined by `S. Akturk et al., Optics Express 12, 4399 (2004) <https://doi.org/10.1364/OPEX.12.004399>`__.
+
+      * ``<laser name>.zeta`` (`float`) optional (default `0.`)
+          Spatial chirp at focus defined by `S. Akturk et al., Optics Express 12, 4399 (2004) <https://doi.org/10.1364/OPEX.12.004399>`__.
+
+      * ``<laser name>.phi2`` (`float`) optional (default `pi/2`)
+          The amount of temporal chirp :math:`\phi^{(2)}` at focus (in the lab frame).
+          Namely, a wave packet centered on the frequency :math:`(\omega_0 + \delta \omega)` will reach its peak intensity at :math:`z(\delta \omega) = z_0 - c \phi^{(2)} \, \delta \omega`.
+          Thus, a positive :math:`\phi^{(2)}` corresponds to positive chirp, i.e., red part of the spectrum in the front of the pulse and blue part of the spectrum in the back.
+          More specifically, the electric field in the focal plane is of the form:
+
+          .. math::
+
+              E(\boldsymbol{x},t) \propto Re\left[ \exp\left(  -\frac{(t-t_{peak})^2}{\tau^2 + 2i\phi^{(2)}} + i\omega_0 (t-t_{peak}) + i\phi_0 \right) \right]
+
+          where :math:`\tau` is given by ``<laser_name>.tau`` and represents the Fourier-limited duration of the laser pulse. Thus, the actual duration of the chirped laser pulse is:
+
+          .. math::
+
+               \tau' = \sqrt{ \tau^2 + 4 (\phi^{(2)})^2/\tau^2 }
+
+         See also the definition in `S. Akturk et al., Optics Express 12, 4399 (2004) <https://doi.org/10.1364/OPEX.12.004399>`__.
+
```

Comment:
Can be removed if there's 1 clear mention of Akturk above. Let's check that it looks good in the final documentation.
```suggestion
```